src/CommitmentsV2/SFA.DAS.CommitmentsV2.Api/Controllers/ApprenticeshipUpdateController.cs

```diff
@@ -0,0 +1,35 @@
+﻿using System.Threading.Tasks;
+using MediatR;
+using Microsoft.AspNetCore.Authorization;
+using Microsoft.AspNetCore.Mvc;
+using SFA.DAS.CommitmentsV2.Api.Types.Responses;
+using SFA.DAS.CommitmentsV2.Application.Queries.GetApprenticeshipUpdate;
+using SFA.DAS.CommitmentsV2.Shared.Interfaces;
+
+namespace SFA.DAS.CommitmentsV2.Api.Controllers
+{
+    [ApiController]
+    [Authorize]
+    [Route("api/apprenticeships/{ApprenticeshipId}")]
```

Comment:
I think the route here should be `api/apprenticeships/{ApprenticeshipId}/updates` or `api/apprenticeships/{ApprenticeshipId}/apprenticeship-updates`?